### PR TITLE
Create Contest UI and integration

### PR DESCRIPTION
**Pull request by ronald-so**
UI for the Create Contest page, still some things that need to be worked on:

- [x] Datepicker (just found out there was one built into Material UI, will try to implement that rather than use other packages)
- [ ] Checkmark when selecting inspirational designs, just like in the mock
- [ ] Pull inspirational designs from S3 bucket (currently hardcoded)

### CODE REVIEW

File: client/src/App.js [hunk 1]
```diff
@@ -15,6 +16,7 @@ function App() {
         <Switch>
           <Route exact path="/" component={LandingPage} />
           <Route path="/login" component={Login} />
+          <Route path="/createcontest" component={CreateContest} />
```
Comment (bonnieli) [inline client/src/App.js]: i think @maziasy 's route naming convention was in snake case like `view-submission`. this should either be `createContest` or `create-contest` to stay consistent :)

File: client/src/pages/CreateContest.jsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+import { SignUpButton } from "./SignUpButton";
+import { CreateContestPaper } from "./CreateContestPaper";
+import { SignInPaper } from "./SignInPaper";
+import { BasicTextField } from "./TextField";
+import { SignInButton } from "./SignInButton";
+import { FormControl, makeStyles, TextField, Typography, Grid } from '@material-ui/core';
+
+const useStyles = makeStyles(theme => ({
+  root: {
+    "margin-top": 5,
```
Comment (bonnieli) [inline client/src/pages/CreateContest.jsx]: you can write snake case for more of a react-y way! `marginTop`

File: client/src/pages/CreateContest.jsx [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    "margin-bottom" : 5,
+    width: 300,
+  },
+  caption: {
+  	"color": theme.default,
```
Comment (bonnieli) [inline client/src/pages/CreateContest.jsx]:
```suggestion
  	color: theme.default
```

File: client/src/pages/CreateContestButton.jsx [hunk 4]
```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { makeStyles, Button, Typography, Grid } from '@material-ui/core';
+
+const useStyles = makeStyles(theme => ({
+  root: {
+    "font-family" : theme.fontFamily,
+  },
+  button: {
+  	"background-color": "black",
+  	"color": theme.bgcolor,
```
Comment (bonnieli) [inline client/src/pages/CreateContestButton.jsx]: linter!

File: client/src/pages/CreateContestButton.jsx [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+    "border-radius": "0",
+    padding: theme.spacing(4, 10)
+  },
+  buttonContainer: {
+  	marginTop: "50px"
```
Comment (bonnieli) [inline client/src/pages/CreateContestButton.jsx]: i see a mix of this snake case and string haha. keep consistent, but sometimes if you are in need to write a bit more complicated CSS, you will have to use strings :)

**Review by bonnieli — COMMENTED** [review]
overall looks really good!

File: client/src/App.js (hunk 1 above)
Comment (ronald-so) [reply client/src/App.js]: changed it to `/create-contest` for consistency!

File: client/src/pages/CreateContest.jsx (hunk 2 above)
Comment (ronald-so) [reply client/src/pages/CreateContest.jsx]: will do!

File: client/src/pages/CreateContest.jsx (hunk 3 above)
Comment (ronald-so) [reply client/src/pages/CreateContest.jsx]: fixed!

**Comment by ronald-so** [comment]
Done with the integration for the most part. Some things that can still be added are client-side validation, timezones, and adding the selected list of inspirational images to the POST request (will have to update models as well though)

**Pull request merged**